messaging.rst

```diff
@@ -187,14 +187,40 @@ Invariants
 Fields
 ^^^^^^
 
+This should match `the Secret message in encoding/messages file <https://github.com/raiden-network/raiden/blob/a19a6c853b55f13725f2545c77b0475cbcc86807/raiden/encoding/messages.py#L113>`_.
```

Comment:
note: Once renamed we would need to change the text here to "Unlock"

Reply:
That falls into my task list (it's even shrinking these days!).